tests/infra/ccf.py

```diff
@@ -265,34 +265,81 @@ def add_users(self, users):
                 "./genesisgenerator", "cert", "--name={}".format(u)
             ).check_returncode()
 
-    def vote_using_majority(self, proposal_id, accept, member_id=1, remote_node=None):
-        member_count = int(len(self.members) / 2 + 1)
-        for i, member in enumerate(self.members):
-            if i >= member_count:
-                break
-            res = self.vote(proposal_id, accept, member, remote_node)
-            if res:
-                break
-
-    def vote(self, proposal_id, accept, member_id=1, remote_node=None):
+    def member_client_rpc(self, member_id, remote_node, *args):
```

Comment:
I am not convinced that we should support two types of `member_client_rpc` calls. AFAICS, we only have this because `memberclient.cpp` does not display the proposals in a json format? 

Since we will probably deprecate the C++ clients in the near future and only return JSON objects, I'd be tempted to change the `display_proposals()` method in `memberclient.cpp` to display a JSON object instead.

Reply:
We'll also need to update the docs accordingly: https://microsoft.github.io/CCF/governance.html#displaying-proposals